torch/csrc/jit/passes/subgraph_rewrite.cpp

```diff
@@ -29,7 +29,7 @@ void SubgraphRewriter::RegisterRewritePattern(
 script::Module SubgraphRewriter::runOnModule(const script::Module& module) {
   nodes_to_delete_.clear();
   for (const auto& m : module.get_methods()) {
-    auto g = m.function().graph();
+    auto g = dynamic_cast<FunctionImpl&>(m.function()).graph();
```

Comment:
I feel like some of these will not always have graphs. For instance, what happens if a module has a builtin method defined on it? We should rewrite these sites to do the right thing, e.g. just not run on non-existing graphs.